android/app/src/components/Button/index.tsx

```diff
@@ -0,0 +1,34 @@
+import {StyleSheet, Text, TouchableOpacity} from 'react-native';
+import React from 'react';
+
+interface Props {
+  toShow: string;
+  isOperation: boolean;
+  handleOnPress: (toShow: string, isOperation: boolean) => void;
+  styles: any;
+}
+
+const Button: React.FC<Props> = ({
+  toShow,
+  isOperation,
+  handleOnPress,
+  styles,
+}) => {
+  return (
+    <TouchableOpacity
+      style={styles}
+      onPress={() => handleOnPress(toShow, isOperation)}>
+      <Text style={stylesButton.text}>{toShow}</Text>
+    </TouchableOpacity>
+  );
+};
+
+const stylesButton = StyleSheet.create({
+  text: {
+    fontSize: 30,
+    textAlign: 'center',
+    color: '#FFF',
+  },
+});
```

Comment:
Realiza un archivo `styles.ts` sobre esta misma carpeta `Button` e importala.

`import styles from './styles.ts'`

Dentro del archivo seria algo como esto:
```suggestion
import { StyleSheet } from 'react-native/types';

const styles = StyleSheet.create({
  text: {
    fontSize: 30,
    textAlign: 'center',
    color: '#FFF',
  },
  buttonBase(Ponle el nombre que quieras): {
   "Las propiedades de estilo que se repiten en todos los usos"
  }
});

export default styles;
```